src/test/java/nextstep/subway/line/domain/SectionsTest.java

```diff
@@ -0,0 +1,35 @@
+package nextstep.subway.line.domain;
+
+import static java.util.Arrays.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import nextstep.subway.station.domain.Station;
+
+class SectionsTest {
+
+    @Test
```

Comment:
지하철 노선과 같은 경우 주석으로 노선의 현황을 표현해주시면 테스트 가독성이 높아질 것 같습니다 😄 
```suggestion
    /**
     *  강남역 - 서울역 - 구파발 - 신촌역 - 대화역
     */
    @Test
```